animal_adoption/models/db.py

```diff
@@ -1,2 +1,162 @@
-from .. import db
+from animal_adoption import db
+from werkzeug.security import generate_password_hash, \
+     check_password_hash
 
+
+adoption_relationship = db.Table(
+    'RelationshipTable',
+    db.Column('adopter_id', db.Integer, db.ForeignKey('AdoptersTable.id_adopters')),
+    db.Column('animal_id', db.Integer, db.ForeignKey('AnimalsTable.id_animals'))
+)
+
+
+class Users(db.Model):
```

Comment:
I was just curious. Which is more common in Flask to use singular or plural for a class name? (User or Users)
I thought a singular form is common but I'm not familiar with Flask/SQL alchemy convention
Feel free to use what you are comfortable with

Reply:
That's a good question. I will check before merging.